combat-trainer.lic

```diff
@@ -1171,6 +1183,13 @@ class AbilityProcess
         next unless !timer || (Time.now - timer).to_i > 30
         game_state.cooldown_timers[name] = Time.now if activate_khri?(@kneel_khri, name)
       end
+    @barb_buffs.each{|name| if Flags["ap-#{name}-expired"] then Flags.reset("ap-#{name}-expired"); start_barb_buff(name); end}
+  end
+
+  def start_barb_buff(name)
+    ability_data = get_data('spells').barb_abilities[name]
+    DRC.bput("#{ability_data['start_command']}", "#{ability_data['activated_message']}", 'But you are already', 'Your inner fire lacks')
```

Comment:
Yeah, that's sort of the way it's working now. We could set a timer for all buffs to stop being applied for 5 minutes, or just let it work like it is going to now. It's up to the person to determine how much IF they can maintain. If you want a timer, how do you suggest we go about doing that?